src/pages/getInvolved.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import '../styles/getInvolved.css';
+
+function GetInvolved() {
```

Comment:
I would try and following the Page naming convention you had on the other page modules here. Something like GetInvolvedPage or similar could work.